sharktank/sharktank/models/punet/tools/import_brevitas_dataset.py

```diff
@@ -162,11 +165,83 @@ def _get_json_tensor(
             dtype=torch.int32,
             disable_saturate=True,
         )
-        bias_quant = bias_quantizer.quantize(bias, name=bias.name)
+        bias_quant = bias_quantizer.quantize(bias, name=bias_name)
         updated_tensors[bias_quant.name] = bias_quant
-        # Spot check that things look sane.
-        bias_dequant = bias_quant.unpack().dequant()
-        bias_diff = bias.as_torch() - bias_dequant
+
+    # If dealing with a fused QKV layer, then we need to split the weight as
+    # Brevitas concats it along the [n * input_dim, output_dim] axis, where
```

Comment:
Typo in comment, should read: `Brevitas concats it along the [n * output_dim, input_dim] axis, where`